src/logger/winston-logger.js

```diff
@@ -1,11 +1,38 @@
+import globalEnv from '../global-env'
 import winston from 'winston'
 import { LoggingWinston, express } from '@google-cloud/logging-winston'
 
+/**
+ *  @param {string} releaseBranch - 'master', 'preview', 'staging' or 'release'
+ *  @return {string} kubernetes namespace
+ */
+function getK8sNamespaceName(releaseBranch) {
+  switch(releaseBranch) {
+    case 'release':
+      return 'production'
+    default:
+      return releaseBranch
```

Comment:
Is preview, master namespace available?

Reply:
Yes, they are available on stackdriver logging.
However, in theory, `master` release branch won't exist on production environment .

Reply:
I don't think the preview namespace is available. In keystone-cluster, only `default` namespace is used?

Reply:
It's a good question.
I think I should move `preview` release branch logs to `keystone-cluster`.

Reply:
BTW, keystone-cluster currently stays on kubernetes1.12 which does not offer the latest stackdriver logging mechanism. If our plan does not include the keystone-cluster migration, this will block the preview branch from release.

Reply:
I will use default resource type (legacy stackdriver used) for `preview` release branch.

Reply:
> BTW, keystone-cluster currently stays on kubernetes1.12 which does not offer the latest stackdriver logging mechanism. If our plan does not include the keystone-cluster migration, this will block the preview branch from release. I plan to have the migration to GKE 1.14 with private cluster setup and Identity aware proxy.

Sure, we can plan the migration.
After migration, we can update the `resource` object here to upload the logs with new stackdriver logging API.